bin/client-eth/Cargo.toml

```diff
@@ -1,68 +1,33 @@
 [workspace]
 [package]
 name = "openvm-client-eth"
-description = ""
+description = "OpenVM Reth Guest Program"
+version = "0.3.0"
```

Comment:
we should start updating the version. I put `0.3.0` because we've already changed the input format once majorly so I consider current `main` more like 0.2.0